python/metatensor-operations/metatensor/operations/contiguous.py

```diff
@@ -0,0 +1,91 @@
+from . import _dispatch
+from ._backend import (
+    Labels,
+    TensorBlock,
+    TensorMap,
+    torch_jit_is_scripting,
+    torch_jit_script,
+)
+
+def is_contiguous(tensor: TensorMap) -> bool:
+    r"""
```

Comment:
```suggestion
    """
```
We don't need this r-formatting here as there is no math i the docstring